srcpkgs/freexl/template

```diff
@@ -1,17 +1,17 @@
 # Template file for 'freexl'
 pkgname=freexl
-version=1.0.5
+version=1.0.6
 revision=1
 build_style=gnu-configure
 short_desc="Library to extract valid data from within an Excel (.xls) spreadsheet"
 maintainer="Giuseppe Fierro <gspe@ae-design.ws>"
-license="MPL-1.1,GPL-2,LGPL-2.1"
+license="MPL-1.1,GPL-2-or-later,LGPL-2.1-or-later"
 homepage="https://www.gaia-gis.it/fossil/freexl/index"
 distfiles="http://www.gaia-gis.it/gaia-sins/${pkgname}-${version}.tar.gz"
-checksum=3dc9b150d218b0e280a3d6a41d93c1e45f4d7155829d75f1e5bf3e0b0de6750d
+checksum=3de8b57a3d130cb2881ea52d3aa9ce1feedb1b57b7daa4eb37f751404f90fc22
 
 freexl-devel_package() {
-	depends="${makedepends} ${sourcepkg}-${version}_${revision}"
+	depends="${sourcepkg}-${version}_${revision}"
```

Comment:
Shouldn't it be `>=` ?

Reply:
shlib bumps would make `>=` wrong, I think. Not completely sure though

Reply:
Manual.md says `>=`

Reply:
In practice it probably doesn't matter much, but `>=` is best practice.

Holding the `-devel` package while still upgrading `${sourcepkg}` will, for example, still allow local static linking against an old version without risking breakage of dependants of `${sourceppkg}`. Sure, this is a contrived example, but why prohibit this behavior for no real benefit?

Reply:
Ok, fair points, I get it

Reply:
Done